website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -1,110 +1,122 @@
 ---
-title: Installing the dbt Cloud CLI (Alpha)
+title: Install dbt Cloud CLI 
 id: cloud-cli-installation
 description: "Instructions for installing and configuring dbt Cloud CLI"
+pagination_next: "docs/cloud/configure-cloud-cli"
 ---
 
-:::warning Alpha functionality 
+:::info Beta functionality
 
-The following installation instructions are for the dbt Cloud CLI, currently in Alpha (actively in development and being tested). 
-
-These instructions are not intended for general audiences at this time. 
+The dbt Cloud CLI is currently in [public beta](/docs/dbt-versions/product-lifecycles#dbt-cloud). Share feedback or request features you'd like to see on the [dbt community Slack](https://getdbt.slack.com/archives/C05M77P54FL).
 
 ::: 
 
-## Installing dbt Cloud CLI
+dbt Cloud natively supports developing using a command line (CLI), empowering team members to contribute with enhanced flexibility and collaboration. The dbt Cloud CLI allows you to run dbt commands against your dbt Cloud development environment from your local command line.
+
+dbt commands are run against dbt Cloud's infrastructure and benefit from:
+
+* Secure credential storage in the dbt Cloud platform.
+* Automatic deferral of build artifacts to your Cloud project's production environment. 
+* Speedier, lower cost builds.
+* Significant platform improvements, to be released over the coming months.
+
+Note, the dbt Cloud CLI is a separate tool powered by [dbt Cloud](/docs/cloud/about-cloud/dbt-cloud-features). It differs from [dbt Core](https://github.com/dbt-labs/dbt-core), which provides dbt through its open-source command-line tool.
 
```

Comment:
greg to verify commands are the same (maybe a few not needed in cloud cli, like dbt init)

Reply:
Nearly all are the exact same. The only difference I know of, as of now, is that you need to quote selection syntax - https://dbtlabs.atlassian.net/browse/CLI-140

Reply:
Nearly all are the exact same. The only difference I know of, as of now, is that you need to quote selection syntax - https://dbtlabs.atlassian.net/browse/CLI-140

Reply:
gotcha! i've addressed this in the main pr: https://github.com/dbt-labs/docs.getdbt.com/pull/4173